sdk/keyvault/keyvault/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+cmake_minimum_required (VERSION 3.0)
```

Comment:
any reason not to just use 3.15.1 since this is a brand new repo?

Reply:
No, I'm just trying to integrate CMake with AzureDevOps. I will update to the latest template as soon as it merged to master

Reply:
BTW, it looks like, AzureDevOps Ubuntu 16 has a CMake version 3.12. We will need either to use 3.12 or install 3.15 with some kind of `apt-get`.